R/cfbd_betting.R

```diff
@@ -136,10 +137,27 @@ cfbd_betting_lines <- function(game_id = NULL,
       # Get the content and return it as data.frame
       df <- res %>%
         httr::content(as = "text", encoding = "UTF-8") %>%
+        stringr::str_replace_all("NaN", 'null') %>%
         jsonlite::fromJSON(flatten = TRUE) %>%
         purrr::map_if(is.data.frame, list) %>%
         dplyr::as_tibble() %>%
-        tidyr::unnest("lines")
+        tidyr::unnest("lines") %>%
+        dplyr::mutate(
```

Comment:
these need to be `.data$overUnder`, `.data$spread`, `.data$formattedSpread` on the RHS inside the `dplyr::case_when`. Please make these changes and it's approved.

Reply:
resolved by [60b1fb2](https://github.com/sportsdataverse/cfbfastR/pull/96/commits/60b1fb27e93ecab68fa01a433dc5fd33a9282c2e)